workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/WorkflowsMgtApi.java

```diff
@@ -21,40 +24,53 @@
 import org.springframework.web.servlet.mvc.method.annotation.ResponseBodyEmitter;
 import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
 
+import java.util.List;
+
 @Api("Api to manage workflow swadl lifecycle")
 public interface WorkflowsMgtApi {
   String X_MANAGEMENT_TOKEN_KEY = "X-Management-Token";
 
-  @ApiOperation("Validate and deploy a workflow SWADL content.")
+  @ApiOperation("Validate and create a new workflow SWADL content.")
```

Comment:
it's still a deploy for me

Reply:
if the publish is set to false, it won't deploy the workflow. So deploy is not that right, wdyt?

Reply:
changed to saveAndDeploy